attachments/templates/attachments/edit_attachment.html

```diff
@@ -0,0 +1,17 @@
+{% extends "site_base.html" %}
```

Comment:
It should be noted that the changed/additions to templates are only for tests. We don't actually use these templates in practice.

Reply:
hmmm this looks confusing from the start.......

Reply:
Well, `edit_attachment.html` didn't exist (it's supposed to). And `site_base.html` doesn't exist (it's supposed to).

Reply:
why is it suppose to exist? i dont see it being required anywhere

Reply:
Both `create_attachment.html` and `edit_attachment.html` extend from `site_base.html` and both of those are used when GETing their respective views. I added tests around that in this PR. In practice however, PolicyStat passes in a different template to use.